arraycheck_rdata.wdl

```diff
@@ -10,50 +10,36 @@ task stuart_arraycheck_rdata {
 		Boolean fastfail = false  # should we exit as soon as we get our first mismatch?
 	}
 
-	Int test_size = 2*ceil(size(test, "GB"))
-	Int truth_size = 2*ceil(size(truth, "GB"))
+	Int test_size = ceil(size(test, "GB"))
+	Int truth_size = ceil(size(truth, "GB"))
 	Int finalDiskSize = test_size + truth_size + 1
 
 	command <<<
 
-	# the md5 stuff pulls from the files in /inputs/
-	# the Rscript pulls from the copied files
-
 	failflag=false
 	for j in ~{sep=' ' test}
 	do
-		
-		# md5
 		md5sum ${j} > sum.txt
 		test_basename="$(basename -- ${j})"
 
-		# R
-		cp ${j} .
-		mv ${test_basename} "testcopy_${test_basename}"
-
 		for i in ~{sep=' ' truth}
 		do
 			truth_basename="$(basename -- ${i})"
+
+			# We assume the test file and truth file have the same basename
+			# Due to how WDL input work, they have a different absolute path
```

Comment:
* `inputs`

Reply:
This sort of thing keeps happening. I code in Sublime Text, which can [spell check in comments](https://forum.sublimetext.com/t/spell-check-for-comments-only/21092/3), but it doesn't seem to recognize WDL comments as comments ("toggle comment" doesn't work even with the Broad's WDL syntax highlighter active and it seems most people just have issues with the keystroke but I'm using the menu). Still, there is a grammar check package out there, so if I can get it recognize comment scope, I should be able to stop this from happening...

Before I go down the force-sublime-to-recognize-a-comment rabbithole, do you know if there's something like lint for spelling/grammar comments? That might be easier than trying to wrangle Sublime. I've had issues with Package Control anyway.

Reply:
Only the blinding mental flare of the English language wrought into a shape that was never meant to exist.